### ISSUE TITLE
An error in ssim?

### ISSUE THREAD

**Issue by yuningzhang2022** [jzyustc/MBRS#5]
There might be an error in **train.py**: 
"_TypeError: ssim() got an unexpected keyword argument 'reduction', (line 98, in train.py), ssim = 1 - 2 * kornia.losses.ssim(encoded_images.detach(), images, window_size=5, reduction="mean")_. "
It seems that "ssim" should be **"ssim_loss"**. Is it right?

**Comment by jzyustc** [comment]
Hi, the TypeError is caused by the different kornia version. It will be solved if you use kornia 0.3.0.

**Comment by yuningzhang2022** [comment]
Thank you very much!
